cli/wizard/commands/list-squid-token/index.ts

```diff
@@ -115,69 +115,95 @@ export async function listSquidToken() {
   const tokenListPath = path.resolve(process.cwd(), ...relativePath);
 
   await spinner("Creating PR...", async () => {
-    await $`git checkout -b feat/add-${newTokenConfig.symbol}-token`;
+    await $`git checkout -b feat/add-${newTokenConfig.prettySymbol}-token`;
     await $`git add ${tokenListPath}`;
-    await $`git commit -m "feat: add ${newTokenConfig.symbol} token"`;
+    await $`git commit -m "feat: add ${newTokenConfig.prettySymbol} token"`;
     await $`git push -u origin HEAD`;
   });
 }
 
 export type InterchainTokenInfo = {
   tokenId: string;
   tokenAddress: string;
+  tokenManagerAddress: string;
+  tokenManagerType: string;
   isOriginToken: boolean;
   isRegistered: boolean;
   chainId: number;
   axelarChainId: string;
   chainName: string;
   wasDeployedByAccount?: boolean;
-  kind: "canonical" | "standardized";
+  kind: "canonical" | "interchain" | "customInterchain";
 };
 
 export type InterchainTokenSearchResult = InterchainTokenInfo & {
   matchingTokens: InterchainTokenInfo[];
 };
 
-export type RemoteInterchainToken = {
+export type RemoteInterchainTokenApiResponse = {
+  name: string;
+  symbol: string;
   axelarChainId: string;
   tokenAddress: string;
+  tokenManagerAddress: string;
+  tokenManagerType: string;
   deploymentStatus: string;
   deploymentTxHash: string;
 };
 
-export type InterchainTokenDetails = {
+export type InterchainTokenDetailsApiResponse = {
   kind: string;
```

Comment:
```suggestion
   tokenType: string;
```
Would be good to use the same term everywhere

Reply:
it's `kind` in the downstream response. we'll have to change the database table and the API. can consider a refactor of that later on